src/renderer/containers/QuickStart/DolphinMigrationStep.tsx

```diff
@@ -0,0 +1,152 @@
+/** @jsx jsx */
+import { ipc_copyDolphin } from "@dolphin/ipc";
+import { DolphinLaunchType } from "@dolphin/types";
+import { css, jsx } from "@emotion/react";
+import styled from "@emotion/styled";
+import Box from "@material-ui/core/Box";
+import Container from "@material-ui/core/Container";
+import FormHelperText from "@material-ui/core/FormHelperText";
+import * as fs from "fs-extra";
+import path from "path";
+import React from "react";
+import { Controller, useForm } from "react-hook-form";
+
+import { Button } from "@/components/FormInputs/Button";
+import { PathInput } from "@/components/PathInput";
+import { useDesktopApp } from "@/lib/hooks/useQuickStart";
+
+import { QuickStartHeader } from "./QuickStartHeader";
+
+const MIGRATE_TEXT = "Migrate Dolphin";
+const SKIP_TEXT = "Skip Migration";
+
+export const MigrateDolphinStep: React.FC = () => {
+  const [netplayMigration, setNetplayMigration] = React.useState(true);
+  const oldDesktopAppPath = useDesktopApp((store) => store.path);
+  const setExists = useDesktopApp((store) => store.setExists);
+  const oldDesktopDolphin = path.join(oldDesktopAppPath, "dolphin");
+
+  const migrateNetplayDolphin = async () => {
+    await ipc_copyDolphin.renderer!.trigger({ dolphinType: DolphinLaunchType.NETPLAY, userPath: netplayPath });
+    setNetplayMigration(false);
+  };
+
+  const deleteOldDesktopAppFolder = async () => {
+    await fs.remove(oldDesktopAppPath);
+    setExists(false);
+  };
+  const migratePlaybackDolphin = async () => {
+    await ipc_copyDolphin.renderer!.trigger({ dolphinType: DolphinLaunchType.PLAYBACK, userPath: oldDesktopDolphin });
+    await deleteOldDesktopAppFolder();
+  };
+
+  const {
+    handleSubmit,
+    watch,
+    control,
+    setValue,
+    formState: { errors },
+  } = useForm<{ netplayPath: string }>();
+  const netplayPath = watch("netplayPath");
+
+  const onNetplaySubmit = handleSubmit(migrateNetplayDolphin);
+
+  return (
+    <Box display="flex" flexDirection="column" flexGrow="1">
+      <Container>
+        <QuickStartHeader>Migrate Dolphin</QuickStartHeader>
+        {netplayMigration && (
```

Comment:
I feel like this "step" is really two steps since it handles both netplay migration and playback migration. I think I would prefer this to either be separate steps, with separate quick start components, or a single component where you can see both options all the time. e.g. check boxes for what you want to migrate: checkbox for netplay, and a checkbox for playback, where if netplay is checked a path input box shows That way it seems more like a single cohesive step.